bin/scrape_software_versions.py

```diff
@@ -11,8 +11,15 @@
     'fastp': ['v_fastp.txt', r"fastp (\S+)"],
     'megahit': ['v_megahit.txt', r"MEGAHIT v(\S+)"],
     'metabat': ['v_metabat.txt', r"version (\S+)"],
-    'checkm': ['v_checkm.txt', r"CheckM v(\S+)"],
-    'refinem': ['v_refinem.txt', r"RefineM v(\S+)"]
+    #'checkm': ['v_checkm.txt', r"CheckM v(\S+)"],
```

Comment:
Why keeping it here as a comment when you removed it from the `environment.yml`?

Reply:
I believe I removed now all checkm/refinem code